internal/rukpak/source/unpacker.go

```diff
@@ -59,15 +59,6 @@ type Result struct {
 type State string
 
 const (
-	// StatePending conveys that a request for unpacking a bundle has been
-	// acknowledged, but not yet started.
-	StatePending State = "Pending"
-
-	// StateUnpacking conveys that the source is currently unpacking a bundle.
-	// This state should be used when the bundle contents are being downloaded
-	// and processed.
-	StateUnpacking State = "Unpacking"
-
```

Comment:
This is somewhat outside of the scope of the intent of this PR, but while I was identifying where to set the `Progressing` condition I noted that we don't ever expect these unpack states from being returned. I figured it is a small enough change that it made sense to include it as part of this PR.